web-tester-product/pom.xml

```diff
@@ -12,6 +12,8 @@
   <artifactId>web-tester-product</artifactId>
   <version>13.2.0-SNAPSHOT</version>
   <packaging>pom</packaging>
+  
+  <name>web-tester-product</name>
```

Comment:
seems like Central Portal has a more rigid validation: if we don't supply a name the release-deployment fails.
![central-portal-names](https://github.com/user-attachments/assets/5151e2d4-583b-4fe8-bc69-1fcca2d0630f)